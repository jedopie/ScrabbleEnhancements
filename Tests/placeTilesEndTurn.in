2
Tests/emptyGame.save
place U at A1
place T at A2
done
