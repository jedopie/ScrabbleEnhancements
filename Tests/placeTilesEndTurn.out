Welcome to Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits (Show student information)
4. Quit

> 
Enter the filename from which load a game
> 
Scrabble Game Successfully Loaded!

JOHN, it's your turn
Score for JOHN: 0
Score for JED: 1
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| R |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
D-2,J-8,U-1,R-1,E-1,T-1,U-1
> > > 
JED, it's your turn
Score for JOHN: 2
Score for JED: 1
   0   1   2   3   4   5   6   7   8   9  10  11  12  13  14
  ------------------------------------------------------------
A| R | U | T |   |   |   |   |   |   |   |   |   |   |   |   | 
B|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
C|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
D|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
E|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
F|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
G|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
H|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
I|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
J|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
K|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
L|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
M|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
N|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 
O|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   | 

Your hand is
I-1,L-1,O-1,W-4,L-1,M-3,O-1
> 